packages/status/tests/php/test-status.php

```diff
@@ -503,6 +511,10 @@ public function get_site_suffix_examples() {
 				'https://localhost',
 				'localhost',
 			),
+			'double_domain'    => array(
+				'https://example.org/http://example.com',
+				'example.org',
```

Comment:
Or should it be "example.org::http:::::example.com", to match what happens with the test on lines 503 and 504?

Hopefully in practice no one really does have "http://" in the path part of their site...

Reply:
Yeah that would make the method a bit simpler. I wasn't sure what the desired out come was and if `example.org::http:::::example.com` was acceptable

Reply:
I'm pretty good at thinking of edge cases, but I don't always know what should be done for them. 😀

If I had to pick, I'd go with the straightforward `example.org::http:::::example.com` until someone else says different. But it's your PR, so you should pick.

Reply:
Ha! Lets keep it simple then :)

Reply:
6371092